src/gracenotegroup.ts

```diff
@@ -14,19 +14,40 @@ import { Beam } from './beam';
 import { StaveTie } from './stavetie';
 import { TabTie } from './tabtie';
 import { StaveNote } from './stavenote';
+import { Note } from './note';
 
-// To enable logging for this class. Set `Vex.Flow.GraceNoteGroup.DEBUG` to `true`.
-function L(...args) {
+export interface GraceNoteGroupFormatOptions {
```

Comment:
if it is for internal use only, should it be exported?

Reply:
Hi good to see you! I believe that anyone using the `format` will require this definition.

Reply:
I'm not a fan of these single-field structures, it makes it harder to read the code as you jump around through types. Let's just use inline types here.